staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go

```diff
@@ -119,6 +122,10 @@ func (s *ServerRunOptions) Validate() []error {
 		errors = append(errors, fmt.Errorf("--min-request-timeout can not be negative value"))
 	}
 
+	if s.MinimalShutdownDuration < 0 {
+		errors = append(errors, fmt.Errorf("--minimum-shutdown-duration can not be negative value"))
+	}
+
```

Comment:
seem we have some test case about `Validate()` in test file, add the a case in UT better?

Reply:
done